Lib/test/test_asyncio/test_tasks.py

```diff
@@ -91,6 +92,17 @@ def setUp(self):
         self.loop.set_task_factory(self.new_task)
         self.loop.create_future = lambda: self.new_future(self.loop)
 
+    def test_task_del_collect(self):
+        class Evil:
+            def __del__(self):
+                gc.collect()
+
+        @asyncio.coroutine
+        def run():
+            return Evil()
+
+        yield from asyncio.gather(*[asyncio.Task(run()) for _ in range(100)])
```

Comment:
This test won't run. `yield from` makes `test_task_del_collect()` a generator and test suite doesn't know how to run them.

Please change to 

```python
self.loop.run_until_complete(
    asyncio.gather(*[asyncio.Task(run()) for _ in range(100)]))
```